ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/CommandGenerator.scala

```diff
@@ -3,30 +3,10 @@
 
 package com.daml.ledger.api.benchtool.submission
 
-import com.daml.ledger.api.benchtool.config.WorkflowConfig.{
-  FibonacciSubmissionConfig,
-  FooSubmissionConfig,
-  SubmissionConfig,
-}
 import com.daml.ledger.api.v1.commands.Command
-import com.daml.ledger.client.binding.Primitive
 
 import scala.util.Try
 
 trait CommandGenerator {
   def next(): Try[Seq[Command]]
 }
-
-object CommandGenerator {
-  def apply(
```

Comment:
This is an extension point for additional generators, in-lining this logic means that to add another generator requires editing and extending `LedgerApiBenchTool` which is possibly already too long and complicated.